### PR TITLE
Netperf: Add argument for namespace cleanup

### PR DESCRIPTION
**Pull request by mkoderer**

### CODE REVIEW

File: network/benchmarks/netperf/launch.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ var (
 	tag            string
 	kubeConfig     string
 	netperfImage   string
+	cleanup_only   bool
```
Comment (gonzolino) [inline network/benchmarks/netperf/launch.go]: You should rename `cleanup_only` to `cleanupOnly` in order to make the linter happy.

File: network/benchmarks/netperf/launch.go [hunk 2]
```diff
@@ -73,6 +74,8 @@ func init() {
 	flag.StringVar(&netperfImage, "image", "sirot/netperf-latest", "Docker image used to run the network tests")
 	flag.StringVar(&kubeConfig, "kubeConfig", "",
 		"Location of the kube configuration file ($HOME/.kube/config")
+	flag.BoolVar(&cleanupOnly, "cleanup", false,
```
Comment (bowei) [inline network/benchmarks/netperf/launch.go]: Let change the description to be more explicit: "Run the cleanup resources phase only (use this flag to clean up orphaned resources from a test run)"

**Pull request merged**